<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QWidget" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="delAllButton">
         <property name="text">
          <string>delAllButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeCurrentItem">
         <property name="text">
          <string>changeCurrentItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeColor">
         <property name="text">
          <string>changeColor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawOneGraph">
         <property name="text">
          <string>drawOneGraph</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectDir">
         <property name="text">
          <string>selectDir</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutGraph"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
